### DISCUSSION TITLE
CsvHelper

### DISCUSSION THREAD

**Discussion by markagama** [oqtane/oqtane.framework#4523] [General] [answered]
I've been away from Oqtane for a while and have pulled the latest dev, compiled and run and I get an error message about CsvHelper not being available.

I would appreciate some guidance on why I would get this error message?

**Comment by markagama** [comment]
solved my own problem by deleting the repo and cloning again.

**Reply by markagama** [reply]
woops, spoke to quickly, added the code for syncfusion blazor and added csvhelper to module, client, shared and server, yet oqtane does not run - indicates missing csvhelper.
Any ideas appreciated.

**Reply by markagama** [reply, marked as answer]
silly me, found that the line to copy dll was missing in debug.cmd

**Reply by leigh-pointer** [reply]
There are syncfusion templates available in the marketplace!

**Reply by markagama** [reply]
thx Leigh, I'll register and take a look at what is available.
